hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -811,6 +863,34 @@ func reconcileUserDataSecret(userDataSecret *corev1.Secret, nodePool *hyperv1.No
 	return nil
 }
 
+func reconcileTunedConfigMap(tunedConfigMap *corev1.ConfigMap, nodePool *hyperv1.NodePool, tunedConfig string) error {
+	tunedConfigMap.Immutable = k8sutilspointer.BoolPtr(false)
+	if tunedConfigMap.Annotations == nil {
+		tunedConfigMap.Annotations = make(map[string]string)
+	}
+	if tunedConfigMap.Labels == nil {
+		tunedConfigMap.Labels = make(map[string]string)
+	}
+
+	tunedConfigMap.Labels[tunedConfigMapLabel] = "true"
+	tunedConfigMap.Annotations[nodePoolAnnotation] = client.ObjectKeyFromObject(nodePool).String()
+
+	if tunedConfigMap.Data == nil {
+		tunedConfigMap.Data = map[string]string{}
+		tunedConfigMap.Annotations[TokenSecretTokenGenerationTime] = time.Now().Format(time.RFC3339Nano)
```

Comment:
why do we need TokenSecretTokenGenerationTime?

Reply:
My original thinking for keeping it there was that it would be nice to see when the ConfigMap was last updated. Outside of debugging purposes it's probably not necessary, so I have removed it.